src/store/reducers/index.js

```diff
@@ -118,6 +129,34 @@ const reducer = (state = initialState, action) => {
         postVideoError: true,
       };
 
+    case GET_VIDID_START:
+      return {
+        ...state,
+        getVideosStart: true,
+      };
+
+    case GET_VIDID_SUCCESS:
+      let actp = action.payload;
+      console.log(
+        "reducer--GET_VIDID_SUCCESS case",
+        { actp },
+        "actp.video.id",
+        actp.video.id
+      );
+      return {
+        ...state,
+        getVidIdStart: false,
+        getVidIdError: false,
+        videoList: action.payload.videos,
```

Comment:
I don't think we should use videoList here, that state is specifically meant for listing all videos in our current logic. We should create a new state for a single video loaded.